Cargo.toml

```diff
@@ -58,14 +58,14 @@ flat_map = "0.0.4"
 ordermap = "0.3"
 mio = "0.5"
 futures = "0.1"
+futures-cpupool = "0.1"
```

Comment:
Why move it around?

Reply:
they are related now.

Reply:
?

Reply:
I just grouped them together.. better ordered.